dev_guide/deployments.adoc

```diff
@@ -829,3 +924,178 @@ spec:
   serviceAccount: <service_account>
   serviceAccountName: <service_account>
 ----
+
+[[advanced-deployment-strategies]]
+=== Advanced deployment strategies
+
+=== Blue-Green Deployment
+
+Blue-Green deployments involve running two versions of an application at the same time and moving production traffic from the old version to the new version ([more about blue-green deployments](http://martinfowler.com/bliki/BlueGreenDeployment.html)). There are several ways to implement a blue-green deployment in {product-title}.
```

Comment:
We don't point from the description of the strategy to its example in any of the other descriptions and I don't think we need to (you just need to scroll down and finish reading about the strategy you are reading:))